qiskit_nature/second_q/mappers/qubit_converter.py

```diff
@@ -87,15 +88,18 @@ def __init__(
                 results which can occur when operator terms are not consistently ordered.
                 This is disabled by default, because in practice the Pauli-terms will be grouped
                 later on anyways.
-        """
 
+        Raises:
+            ValueError: If the mapper is a ``TaperedQubitMapper``.
+        """
+        if isinstance(mapper, TaperedQubitMapper):
+            raise ValueError(f"{type(mapper)} is not supported by the converter.")
```

Comment:
```suggestion
        if isinstance(mapper, TaperedQubitMapper):
            raise ValueError(
                "The TaperedQubitMapper is not supported by the QubitConverter. If you want to using tapering "
                "please either use the tapering built directly into the QubitConverter (see its documentation) "
                "or use the TaperedQubitMapper standalone (recommended)."
            )

```
I think this will be a bit more helpful for the user. I did not pay attention to line wrapping while writing this in the suggestion on Github so re-wrapping may be necessary.